web/router.ex

```diff
@@ -92,6 +91,7 @@ defmodule CodeCorps.Router do
 
     post "/token", TokenController, :create
     post "/token/refresh", TokenController, :refresh
+    post "/password/reset", PasswordResetController, :reset_password
```

Comment:
This is to mimic the ember route, I'm guessing? I'm ok with the change. Really, I don't think there's a right/wrong way to name things here.